test/core/MakerPosthook.t.sol

```diff
@@ -300,9 +300,9 @@ contract MakerPosthookTest is MangroveTest, IMaker {
     expectFrom($(mgv));
     emit OfferRetract(olKey.hash(), $(this), ofr, true);
     expectFrom($(mgv));
-    emit Credit($(this), 10699440000000000);
+    emit Credit($(this), 10697680000000000);
     expectFrom($(mgv));
-    emit OfferFail(olKey.hash(), $(tkr), ofr, 1 ether, 1 ether, 8500560000000000, "mgv/makerRevert");
+    emit OfferFail(olKey.hash(), $(tkr), ofr, 1 ether, 1 ether, 8502320000000000, "mgv/makerRevert");
```

Comment:
I'm surprised that the penalty is higher now?
Must mean an increase in gas, but I thought this was merely a refactor of constants.

It's negligable, so not important

Reply:
I thought I knew why (change in error message length), but your comment prompted me to check and I was wrong. I drilled it down to the function name change in `MgvView` from `level3` to `root`, which I assume modified the shape of the binary tree solidity descends when it matches a selector against a bytecode entry point.